src/snowflake/snowpark/functions.py

```diff
@@ -750,6 +750,60 @@ def covar_samp(column1: ColumnOrName, column2: ColumnOrName) -> Column:
     return builtin("covar_samp")(col1, col2)
 
 
+def create_map(*cols: Union[ColumnOrName, Union[List[ColumnOrName], Tuple[ColumnOrName, ...]]]) -> Column:
+    """Transforms multiple column pairs into a single map :class:`~snowflake.snowpark.Column` where each pair of
```

Comment:
```suggestion
def create_map(*cols: Union[ColumnOrName, Iterable[ColumnOrName]]) -> Column:
    """Transforms multiple column pairs into a single map :class:`~snowflake.snowpark.Column` where each pair of
```

I think this type hint suits better in this case.

Reply:
You're right! I made the change on commit [989464e](https://github.com/snowflakedb/snowpark-python/pull/1204/commits/989464e5b0f8894ab228ccb5c0717a51deff0dd1)